schema/encoding/jsonschema.go

```diff
@@ -139,6 +140,13 @@ func schemaToJSONSchema(w io.Writer, s *schema.Schema) (err error) {
 		if ew.err == nil {
 			ew.err = validatorToJSONSchema(w, field.Validator)
 		}
+		if field.Default != nil {
+			b, err := json.Marshal(field.Default)
+			if err != nil {
+				return err
+			}
+			ew.writeFormat(`, "default": %s`, string(b))
```

Comment:
Hmm, it's because we got rid of write, we no longer have a byte interface. Should I re-add it?